src/main/java/ac/grim/grimac/checks/impl/badpackets/BadPacketsH.java

```diff
@@ -0,0 +1,36 @@
+package ac.grim.grimac.checks.impl.badpackets;
+
+import ac.grim.grimac.checks.CheckData;
+import ac.grim.grimac.checks.type.PacketCheck;
+import ac.grim.grimac.player.GrimPlayer;
+import com.github.retrooper.packetevents.event.PacketReceiveEvent;
+import com.github.retrooper.packetevents.protocol.packettype.PacketType;
+import com.github.retrooper.packetevents.wrapper.play.client.WrappedPacketInUseEntity;
+import com.github.retrooper.packetevents.wrapper.play.client.WrapperPlayClientPlayerFlying;
+
+@CheckData(name = "BadPacketsH")
+public class BadPacketsH extends PacketCheck {
+    private boolean swung;
+
+    public BadPacketsH(final GrimPlayer player) {
+        super(player);
+    }
+
+    @Override
+    public void onPacketReceive(PacketReceiveEvent event) {
+        if (event.getPacketType() == PacketType.Play.Client.USE_ENTITY) {
```

Comment:
Should be INTERACT_ENTITY. USE_ENTITY is the property on PacketEvents 1.8
```suggestion
        if (event.getPacketType() == PacketType.Play.Client.INTERACT_ENTITY) {
```